modules/prebid/rulesengine/config/config.go

```diff
@@ -16,9 +16,10 @@ const RulesEngineSchemaFile = "rules-engine-schema.json"
 const RulesEngineSchemaFilePath = "modules/prebid/rulesengine/config/" + RulesEngineSchemaFile
 
 type PbRulesEngine struct {
-	Enabled   bool      `json:"enabled,omitempty"`
-	Timestamp string    `json:"timestamp,omitempty"`
-	RuleSets  []RuleSet `json:"rulesets,omitempty"`
+	Enabled                    bool      `json:"enabled,omitempty"`
+	Timestamp                  string    `json:"timestamp,omitempty"`
+	UpdateTreeFrequencyMinutes int       `json:"updatetreefrequencyminutes,omitempty"`
```

Comment:
Recommend naming this `RefreshRateSeconds` to match other terminology used in this project and to provide the option for seconds, if desired. Despite the name, the earlier `fiveMinutes` is expressed as a duration in seconds.

Should this be a pointer to differentiate between a `0` value (meaning never refresh) vs a default value?

Reply:
Renamed